javascript/javascript3/week3/shoppingCart/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div id = "product-render"> <h1>Render Product</h1></div>
+    <div id = "product-search"><h1>Product Search</h1>
+      <label for="product-name">Enter Product Name:</label><br>
+      <input type="text" id="product-search" name="product-name"><br>
```

Comment:
Id needs to be something unique, you can't use "product_search" for div and input tag. Also "product_name" needs to be defined first.

Reply:
ohh yahh you are right. I have made mistake in naming conversion.. I have updated.